### PR TITLE
Fixes and updates

### PR DESCRIPTION
**Pull request by KmaTTi89**
- fix for test.bat, don't use "pause" command when running script from console
- updated nunit console runner
- updated webdriver lib

### CODE REVIEW

File: test.bat [hunk 1]
```diff
@@ -1,6 +1,16 @@
 @echo off
 
-IF NOT EXIST "%~dp0packages\NUnit.ConsoleRunner.3.6.0\" (ECHO Error: Cannot find NUnit Console Runner. Build the project to restore the packages && PAUSE && EXIT /B 1)
-%~dp0packages\NUnit.ConsoleRunner.3.6.0\tools\nunit3-console.exe %~dp0test\KitchenSink.Tests\bin\Debug\KitchenSink.Tests.dll --noheader --params Browsers=Chrome,Firefox
-pause
+set interactive=1
+echo %cmdcmdline% | find /i "%~0" >nul
+if not errorlevel 1 set interactive=0
+
+rem now I can use %interactive% anywhere
```
Comment (warpech) [inline test.bat]: This comment is redundant here. Can you change it to something more meaningful like `If we are in interactive mode (batch file not started from the command line), pause at the exit`

File: test.bat [hunk 2]
```diff
@@ -1,6 +1,16 @@
...
+
+IF NOT EXIST "%~dp0packages\NUnit.ConsoleRunner.3.6.1\" (ECHO Error: Cannot find NUnit Console Runner. Build the project to restore the packages && PAUSE && EXIT /B 1)
+%~dp0packages\NUnit.ConsoleRunner.3.6.1\tools\nunit3-console.exe %~dp0test\KitchenSink.Tests\bin\Debug\KitchenSink.Tests.dll --noheader --params Browsers=Chrome,Firefox
+
+if _%1_==__ (
```
Comment (warpech) [inline test.bat]: The above line looks redundant. See: http://stackoverflow.com/questions/2309968/what-are-1-and-2-in-batch-files

File: test.bat (hunk 1 above)
Comment (KmaTTi89) [inline test.bat]: Changed

File: test.bat (hunk 2 above)
Comment (KmaTTi89) [inline test.bat]: Fixed

**Pull request merged**